backend/capellacollab/sessions/idletimeout.py

```diff
@@ -45,6 +45,7 @@ async def loop():
                 await run_in_threadpool(terminate_idle_session)
             except BaseException:
                 log.exception("Could not handle idle sessions")
+                return
```

Comment:
```suggestion
            except asyncio.exceptions.CancelledError:
                return
            except BaseException:
                log.exception("Could not handle idle sessions")
```

I'd rather handle it separately - then we also don't get the error message printed during exit. And in the case of an unknown error - the session termination still continues - otherwise it would stop completely.